fastlane/lib/fastlane/actions/ensure_git_branch.rb

```diff
@@ -34,7 +34,7 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :branch,
                                        env_name: "FL_ENSURE_GIT_BRANCH_NAME",
-                                       description: "The branch that should be checked for. String that can be either the full name of the branch or a regex to match",
+                                       description: "The branch that should be checked for. String that can be either the full name of the branch or a regex e.g. '^feature\/.*$' to match",
```

Comment:
Hm, would using \` instead of ' here maybe format the string as code in the docs output?

Reply:
i agree